tests/defaults.yml

```diff
@@ -1,3 +1,19 @@
 # Overwrite parameters here
 
-# parameters: {...}
+parameters:
+  kyverno:
+    additionalClusterRoles:
+      my-name:
+        - apiGroups:
+            - rbac.authorization.k8s.io
+          verbs:
+            - '*'
+          resources:
+            - rolebindings
+    additionalRoleBindings:
+      allowManageRoleBindings:
+        kind: ClusterRole
+        name: kyverno-user-my-name
+      cluster-admin:
```

Comment:
Maybe that's a suboptimal sample if someone sees it and copies it from here? Not sure if kyverno ever should be `cluster-admin`.

Reply:
Went with the `admin` role instead